src/rpft/parsers/sheets.py

```diff
@@ -9,55 +10,79 @@
 from googleapiclient.discovery import build
 
 
-class CSVSheetReader:
+class AbstractSheetReader(ABC):
+    def csv_path(self):
+        return None
```

Comment:
Wouldn't this only ever apply to `CSVSheetReader`? If so, then this cannot really be considered part of the abstraction.

Reply:
Ok. I removed this an am using `isinstance` in the composite reader's `get_sheet` now.